app/models/crusher.rb

```diff
@@ -0,0 +1,18 @@
+class Crusher
+  def self.create_order(session, current_user, cart)
```

Comment:
Unused method argument - `session`. If it's necessary, use `_` or `_session` as an argument name to indicate that it won't be used.